lib/mcp/app/tool.rb

```diff
@@ -77,7 +77,7 @@ def list_tools(cursor: nil, page_size: 10)
         tool_values = tools.values
         start_index = cursor ? cursor.to_i : 0
         paginated = tool_values[start_index, page_size]
-        next_cursor = (start_index + page_size < tool_values.length) ? (start_index + page_size).to_s : ""
+        next_cursor = (start_index + page_size < tool_values.length) ? (start_index + page_size).to_s : nil
```

Comment:
I checked again, and it looks like `nil` is correct.

https://github.com/modelcontextprotocol/specification/blob/6828f3ef6300b25dd2aaff2a2e5e81188bdbd22e/schema/2024-11-05/schema.ts#L306-L312